### PR TITLE
Make stepper tooltips only appear on non-touch devices

### PR DESCRIPTION
**Pull request by Tankenstein**
Tooltips on the stepper on touch devices have caused problems with going back and overall confusion because of their behaviour, so we removed them.

### CODE REVIEW

File: docs/StepperDocs.js [hunk 1]
```diff
@@ -134,8 +140,9 @@ export default class StepperDocs extends Component {
               }))}
               onChange={value => this.goToStep(value)}
               selected={{
-                label: `${this.state.activeStep} - ${this.state.selectedPreset.steps[this.state.activeStep]
-                  .label}`,
+                label: `${this.state.activeStep} - ${
```
Comment (OFFLlNE) [inline docs/StepperDocs.js]: interesting prettier :)

Comment (oliverviljamaa) [reply docs/StepperDocs.js]: A bit weird, but better than before 😀

File: src/stepper/deviceDetection.js [hunk 2]
```diff
@@ -3,16 +3,18 @@ function supportsTouchEvents() {
...
+    'iemobile',
+    'iphone',
+    'ipad',
+    'ipod',
+    'blackberry',
+    'bada',
+  ];
+  const matchString = sampleTouchDevices.map(device => `(${device})`).join('|');
+  const regex = new RegExp(matchString, 'ig');
+  return !!navigator.userAgent.match(regex);
```
Comment (mstaicu) [inline src/stepper/deviceDetection.js]: I would go with `Boolean(..)` for readability.

Comment (oliverviljamaa) [reply src/stepper/deviceDetection.js]: I think `!!` is more popular and therefore better recognized?

**Pull request merged**